frontend.sh

```diff
@@ -93,6 +93,17 @@ build() {
   fi
 }
 
+# Run tasks to build all project assets for production distribution.
+dry_run() {
```

Comment:
I'm not sure `dry_run` is the best descriptor for what this does. `validate_static_files` maybe?